src/unix/bsd/apple/mod.rs

```diff
@@ -1025,6 +1025,30 @@ pub const RTLD_NODELETE: ::c_int = 0x80;
 pub const RTLD_NOLOAD: ::c_int = 0x10;
 pub const RTLD_GLOBAL: ::c_int = 0x8;
 
+pub const _WSTOPPED: ::c_int = 0o177;
+
+f! {
+    pub fn WSTOPSIG(status: ::c_int) -> ::c_int {
+        status >> 8
+    }
+
+    pub fn WSTATUS(status: ::c_int) -> ::c_int {
```

Comment:
This doesn't look like it's a `#define` in `sys/wait.h` for me, but there's a `_WSTATUS`, so maybe this is an internal function?